package.json

```diff
@@ -34,38 +34,36 @@
   ],
   "license": "MIT",
   "dependencies": {
-    "branch-diff": "^1.10.5",
-    "chalk": "^5.0.1",
-    "changelog-maker": "^2.8.0",
+    "branch-diff": "^2.1.0",
+    "chalk": "^5.2.0",
+    "changelog-maker": "^3.2.1",
     "cheerio": "^1.0.0-rc.10",
     "clipboardy": "^3.0.0",
-    "core-validate-commit": "^3.16.0",
+    "core-validate-commit": "^3.18.0",
     "enquirer": "^2.3.6",
-    "execa": "^6.1.0",
-    "figures": "^4.0.1",
+    "execa": "^7.0.0",
+    "figures": "^5.0.0",
     "form-data": "^4.0.0",
-    "ghauth": "^4.0.0",
-    "inquirer": "^8.2.4",
-    "listr2": "^4.0.5",
+    "ghauth": "^5.0.1",
```

Comment:
`ghauth` has switched to promises (with support for callbacks), we should remove the `promisify`. (We could also send upstream changes to switch from `node-fetch` to `undici`, but that's not something blocking this PR).

Reply:
Done.